### PR TITLE
Better Liip-Imagine/Gaufrette media integration

### PR DESCRIPTION
**Pull request by David-Crty**
| Q               | A
| --------------- | -----
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes https://github.com/BitBagCommerce/SyliusCmsPlugin/issues/431  https://github.com/BitBagCommerce/SyliusCmsPlugin/issues/344
| License         | MIT

I have made some fixes about the `liip/gaufrette` integration, I only used the filter already available by sylius.

Please let me know if I need to do more about this MR :pray: 

Because I'm working on a new project, I'm not sure if the change on `setPath` is a breaking change :thinking:

### CODE REVIEW

File: src/Resources/views/Shop/Media/Show/image.html.twig [hunk 1]
```diff
@@ -1,6 +1,12 @@
 <h2>{{ media.name|raw }}</h2>
 
-<img class="ui fluid image" src="{{ media.path }}" alt="{{ media.name }}"
+{% if page.image.path starts with 'data:' %}
```
Comment (rafalmal) [inline src/Resources/views/Shop/Media/Show/image.html.twig:3]: This change throws 500 error on main page.

**Review by rafalmal — CHANGES_REQUESTED** [review]

File: src/Resources/views/Shop/Media/Show/image.html.twig (hunk 1 above)
Comment (rafalmal) [reply src/Resources/views/Shop/Media/Show/image.html.twig:3]: bump

Comment (David-Crty) [reply src/Resources/views/Shop/Media/Show/image.html.twig:3]: Can you show me the 500?

Comment (marekrzytki) [reply src/Resources/views/Shop/Media/Show/image.html.twig:3]: Hi @David-Crty. I also noticed this 500 error on fresh install with default fixtures.

**Comment by senghe** [comment]
I see no response since last month, so I'm closing this PR. Please reopen it if you still need it @David-Crty

**Pull request closed without merging**